os/include/tinyara/pm/pm.h

```diff
@@ -303,9 +303,14 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
-struct pm_wakeup_timer_s {
-	bool use_timer;
-	uint32_t timer_interval;
+enum pm_timer_type_e {
```

Comment:
Lets remove the USE
enum pm_timer_type_e {
  PM_COMMON_ONE_SHOT_TIMER = 1,
  PM_HOLD_STATE_TIMER                 = 2,
  /* More to add */
  PM_TIMER_MAX                              = 8, // Should never reach, if yes, then we have too much case to handle, consider to expand to u16
}